packages/govuk-frontend/src/govuk/errors/index.mjs

```diff
@@ -28,9 +28,17 @@ export class GOVUKFrontendError extends Error {
 export class SupportError extends GOVUKFrontendError {
   name = 'SupportError'
 
-  // eslint-disable-next-line jsdoc/require-jsdoc -- Nothing pertinent to document
-  constructor() {
-    super('GOV.UK Frontend is not supported in this browser')
+  /**
+   * Checks if GOV.UK Frontend is supported on this page
+   *
+   * @param {HTMLElement | null} [$scope] - HTML element `<body>` checked for browser support
+   */
+  constructor($scope = document.body) {
+    super(
+      $scope
+        ? 'GOV.UK Frontend is not supported in this browser'
+        : 'GOV.UK Frontend initialised before the body element (`<body>`) was available'
```

Comment:
I wondered that, but wasn't sure if it would be mandatory for all services?

What about injected scripts?

Reply:
> I wondered that, but wasn't sure if it would be mandatory for all services?

We can't mandate (they can still `defer` or put at the end of the `<body>`), but `type="module"` is really the way we'd like services to load GOV.UK Frontend. We ask them to [consider code splitting](https://release-5-0--govuk-frontend-docs-preview.netlify.app/changes-to-govuk-frontend-v5/#migrating-from-v4-to-v5) in our migration docs for code that cannot run in a `<script type="module">`. That made it seem fine to be more direct here.

> What about injected scripts?

I'm not sure what you mean (imagining the GA snippet that `document.createElement('<script>')` to inject more code), could you give an example please? 😊

Reply:
Yeah either `document.createElement('<script>')` with `script.type = 'module'` or `await import()`

If you're happy with `<script type="module">` I'm sure advanced users will understand

Reply:
Done ✅